src/app/api/services/dot-http-error-manager/dot-http-error-manager.service.ts

```diff
@@ -140,9 +137,11 @@ export class DotHttpErrorManagerService {
     }
 
     private handleBadRequestError(response: Response): boolean {
-
         this.dotDialogService.alert({
-            message: response.json()['message'] || this.dotMessageService.get('dot.common.http.error.400.message'),
+            message:
```

Comment:
`this.getErrorMessage(response)` should be first.

Reply:
done